.env

```diff
@@ -0,0 +1,3 @@
+# Debug the application actions and state into the 
```

Comment:
`.env` file should not be versionned (use the `.env.dist` file for that ;) ).
The app (no matter on which env) should use the `.env` file to compile and run. Considering this, the `.env` file should not be versionned and should be initialized from a deploy / init workflow (eg Makefile, CI).

Reply:
Why, since it's easier to not have the step of "Copy / paste" the env ?

Reply:
I mean, by versionning the `.env` we are winning:

1. We don't need to copy / past the `.env.dist` file (just use a default `.env` file, it becames the `.env.dist` basicaly)
2. Create react app comes with a `.env.local` override system, allowing you to do the same thing as you mentionned.

Reply:
Although overrides seems to be a quick win in the beginning, they end up in a mess in the mid / end of project development. As the `.env` file is the one used by the app, it should not be versionned as its values depends on which env the app is used.
Having a plain `.dist` file for each env (eg `.env.dist` for dev, `.env.preprod.dist` for preprod, `.env.prod.dist` for prod) is way cleaner and avoids possible overrides errors / misconceptions.

Having to copy / paste the dist file is not a big deal IMO as it is part of project initialization (same as deps install). It's still a one-liner :
```bash
$ make .env install-deps # executes both `.env` and `install-deps` targets
```

Reply:
Why does it mess up in the mid / end ?

Reply:
It gets harder to know what is the resolved value of a parameter for a given env when its value is split in multiple files. It's more readable and searchable to have a distinct env file per env with all the values in it.
*Per se*, the env file is dedicated to a single env, so versionning it binds it to multiple env and so is in contradiction with the aim of an env file.